app/controllers/api/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::GreetingsController < ApplicationController
+  def random_greeting
+    random_message = Message.order('RANDOM()').first
+    render json: { greeting: random_message&.content }
+  end
+end
```

Comment:
- [ ] This controller seems to be redundant and not used. your requests are routed to the `get 'random_greeting', to: 'messages#random_greeting'`. I'll suggest you remove this and make use of only the required controller `MessagesController` :wave: :small_airplane: